.cctemplate

```diff
@@ -69,16 +69,6 @@
             "name": "☸️ k8s-nodejs-guestbook",
             "description": "k8s NodeJS guestbook"
         },
-        {
-            "path": "platforms/gke-wa-shop",
```

Comment:
what would be the reason to delete these 2 templates?

Reply:
they don't exist

Reply:
@redvg